src/export/callback.ts

```diff
@@ -0,0 +1,25 @@
+import type {
```

Comment:
Needs Apache License in the header.

Reply:
Thanks for the feedback! I was considering putting them in there, but I initially wasn't sure how many exports I'd be adding and I didn't want to bloat that file too much. But I think this doesn't add too much bloat, although there are some more callbacks I didn't include as I wasn't sure exactly how they should be exported.